content/blog/2020-07-27-shtab-completion-release.md

```diff
@@ -0,0 +1,212 @@
+---
+title: '(Tab) Complete Any Python Application in 1 Minute or Less'
+date: 2020-07-27
+description: |
+  We've made a painless tab-completion script generator for Python applications!
+  Find out how to take advantage of it in this blog post.
+
+descriptionLong: |
+  We've made a painless tab-completion script generator for Python applications!
+  It's called `shtab` and it currently works with `argparse`, `docopt`, and
+  `argopt` to produce `bash` and `zsh` completion scripts.
+  This tool was originally created to help `dvc`, but we realised it could be made
+  more generic and valuable to the world's entire ecosystem of Python CLI
+  applications. Find out how to take advantage of it in this blog post.
+picture: 2020-07-27/tab-py.jpg
+pictureComment: Zero Effort Tab Completion for Python Applications
+author: casper_dcl
+commentsUrl: https://discuss.dvc.org/t/tab-complete-any-python-application-in-1-minute-or-less/455
+tags:
+  - shtab
+  - Release
+  - CLI
+  - Autocomplete
+  - Tab
+  - Completion
```

Comment:
We forgot to review these. Probably too many @shcheklein @casperdcl ? I'd only keep `Release` and maybe `shtab` because no other tag has any posts. Would maybe add `CLI` if it's added retroactively to a few others.

Reply:
Is there a way to see all current tags?

Reply:
I was wondering the same thing: #1657

Reply:
@jorgeorpinel @casperdcl //cc @elleobrien  - tags serve two goals - navigation (and we want to keep them sane for this), and SEO (keywords in meta - I haven't checked this, but I hope it still works or we need to fix this). For SEO we want to include things that people will use to search - "Python CLI autocomplete", etc. Thus we need to have a good balance. No need to optimize tags just for the sake of trying to keep them to the minimum.

Reply:
OK in that case all tags here seem relevant, for SEO that is. Though I should say, I think most search engines pretty much ignore suggested meta keywords and extract them from the content instead. But if the meta keywords match the content, then it may be a nice extra.